project.yaml

```diff
@@ -0,0 +1,149 @@
+targets:
+  bookworm:
+    architectures:
+      x86_64:
+        test_distros:
+          representative:
+            debian-12
+      aarch64:
+        test_distros:
+          representative:
+            debian-12-arm
+  bullseye:
+    architectures:
+      x86_64:
+        test_distros:
+          representative:
+            debian-11
+      aarch64:
+        test_distros:
+          representative:
+            debian-11-arm
+  buster:
+    architectures:
+      x86_64:
+        test_distros:
+          representative:
+            debian-10
+  centos7:
+    architectures:
+      x86_64:
+        test_distros:
+          representative:
+            centos-7
+          exhaustive:
+            rhel-7
+            rhel-7-9-sap-ha
+  focal:
+    architectures:
+      x86_64:
+        test_distros:
+          representative:
+            ubuntu-minimal-2004-lts
+          exhaustive:
+            ubuntu-2004-lts
+      aarch64:
+        test_distros:
+          representative:
+            ubuntu-minimal-2004-lts-arm64
+          exhaustive:
+            ubuntu-2004-lts-arm64
+  jammy:
+    architectures:
+      x86_64:
+        test_distros:
+          representative:
+            ubuntu-minimal-2204-lts
+          exhaustive:
+            ubuntu-2204-lts
+      aarch64:
+        test_distros:
+          representative:
+            ubuntu-minimal-2204-lts-arm64
+          exhaustive:
+            ubuntu-2204-lts-arm64
+  lunar:
+    architectures:
+      x86_64:
+        test_distros:
+          representative:
+            ubuntu-2304-amd64
+          exhaustive:
+            ubuntu-minimal-2304-amd64
+  mantic:
+    architectures:
+      x86_64:
+        test_distros:
+          representative:
+            ubuntu-minimal-2310-lts
+          exhaustive:
+            ubuntu-2310-lts
+      aarch64:
+        test_distros:
+          representative:
+            ubuntu-minimal-2310-lts-arm64
+          exhaustive:
+            ubuntu-2310-lts-arm64
+  rockylinux8:
+    architectures:
+      x86_64:
+        test_distros:
+          representative:
+            rocky-linux-8
+          exhaustive:
+            rhel-8
+            rhel-8-2-sap-ha
+            rhel-8-4-sap-ha
+            rhel-8-6-sap-ha
+    rockylinux9
```

Comment:
this looks like a parse error, weird indent and missing `:`

Reply:
Done